assets/css/style.css

```
@@ -1,3 +1,6 @@
 .hide {
   display: none;
 }
+.done{
```

separate style rules by line breaks for readability

```suggestion
}

.done{
```